src/main/java/org/springframework/data/elasticsearch/client/reactive/ReactiveElasticsearchClient.java

```diff
@@ -417,6 +419,27 @@
 	 */
 	Flux<SearchHit> search(HttpHeaders headers, SearchRequest searchRequest);
 
+	/**
+	 * Execute the given {@link SearchRequest} against the {@literal search} API.
```

Comment:
Isn't `search` correct? I used that because I found this in the elasticsearch [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters.html#search-suggesters-api-request).

> _suggest endpoint has been deprecated in favour of using suggest via _search endpoint. In 5.0, the _search endpoint has been optimized for suggest only search requests.

Reply:
ah, you're right, I forgot that (not using suggests too often)